### PR TITLE
Documented admin settings, general settings and main menu.

### LINKED ISSUE #54: Describe admin settings menu and how it's typically used
opendatakit/collect#1306 - it's not all that clear

This may exist somewhere but I'm not immediately finding it.

### PR DESCRIPTION
**Pull request by troublemagnet**
closes #54  and   #6 

## What is included in this PR?

- Explained the general settings , admin settings and the main menu options.
- Added screenshots wherever needed
- Added notes and warning as needed

### CODE REVIEW

File: collect-settings.rst [hunk 1]
```diff
@@ -1,18 +1,46 @@
...
+- :menuselection:`Send Finalized Form` lists all the finalized forms and gives you an option to send them.
+- :menuselection:`View Sent Form` displays the forms that have been sent.
+- :menuselection:`Get Blank form` lists all available ODK :term:`Collect` :term:`form` :term:`widget` (question types). The details about which are available on :ref:`basic-form-widgets`.
+- :menuselection:`Delete Saved Form` lists all the Saved and Blank Forms and allows you to delete them.
 
 .. _general-settings:
 
-General Settings
------------------
+2. General Settings
```
Comment (adammichaelwood) [inline collect-settings.rst]: don't put numbers in section titles

File: collect-settings.rst [hunk 2]
```diff
@@ -44,15 +77,21 @@ Map settings are included on the :menuselection:`User Interface` settings page.
...
+.. rubric:: Form submission
+
+- :guilabel:`Auto send` When enabled, forms are sent immediately when finalized if the device can connect to the internet. You can specify whether to send over WiFi, cellular data, or both.
+- :guilabel:`Delete after send` When checked, form instances are deleted once they are sent.
 
 .. rubric:: Form filling
 
 - :guilabel:`Default to finalized` When enabled, forms are automatically finalized upon reaching the end of the form. You can opt out of finalizing any form when completing it.
-- :guilabel:`Constraint processing` sets when response constraints are validated: When advancing through the form, or at finalization.
+- :guilabel:`Constraint processing` used to select when response constraints are validated: When advancing through the form i.e. after each swipe, or at finalization.
```
Comment (adammichaelwood) [inline collect-settings.rst]: > When advancing through the form i.e. after each swipe, or at finalization.

Your fix is an improvement, but fewer words would be better:

```
:guilabel:`Constraint processing` used to select when response constraints are validated: After each swipe, or at finalization.
```

File: collect-settings.rst [hunk 3]
```diff
@@ -44,15 +77,21 @@ Map settings are included on the :menuselection:`User Interface` settings page.
...
 
+.. image:: /img/collect-settings/form-management.*
+  :alt: Form Management settings
+  :class: device-screen-vertical
+
 :menuselection:`Form Management` settings handle default behavior when editing, finalizing, and importing forms.
 
-- :guilabel:`Auto send` When enabled, forms are sent immediately when finalized if the device can connect to the internet. You can specify whether to send over WiFi, cell data, or both.
-- :guilabel:`Delete after send` When enabled, form instances are deleted once they are sent.
+.. rubric:: Form submission
```
Comment (adammichaelwood) [inline collect-settings.rst]: I wonder if these should all be subsection headings (`""""""""""`) 
instead of rubric headings

Do you have a thought on that?
I can't remember why I used rubric headings the first time....

File: collect-settings.rst [hunk 4]
```diff
@@ -89,33 +132,36 @@ You cannot edit these:
...
+.. rubric:: Usage data
 
 When enabled, ODK Collect sends usage and error data back to the ODK development team, which helps us improve the application.
 
 Usage data is anonymized.
 
 .. _admin-settings:
 
-Admin Settings
+3. Admin Settings
```
Comment (adammichaelwood) [inline collect-settings.rst]: don't put numbers in section titles

File: collect-settings.rst [hunk 5]
```diff
@@ -125,7 +171,10 @@ You can also import settings from a QR code saved to the device, by selecting :g
 
 .. _user-access-control-settings:
 
-User Access Control Settings
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. rubric:: User Access Control Settings
```
Comment (adammichaelwood) [inline collect-settings.rst]: why was this changed to a rubric?

File: collect-settings.rst [hunk 6]
```diff
@@ -125,7 +171,10 @@ You can also import settings from a QR code saved to the device, by selecting :g
...
+
+This section allows the admin to hide menu items and settings from the user. It contains the following options:
 
-This section allows the admin to hide menu items and settings from the user.
+- :guilabel:`Main Menu Settings` : Allows admin to hide some of the main-meny options from the users.
```
Comment (adammichaelwood) [inline collect-settings.rst]: typo: menu

**Comment by troublemagnet** [comment]
@adammichaelwood Sir I thought rubric heading were more appropriate than sub-heading because - 
1) In terms of formatting, sub headings have a sightly bigger font-size than rubric headings and in this particular case I found the styling of rubric heading to be more appropriate.
2)  Also referring to http://www.sphinx-doc.org/en/stable/markup/para.html, I felt that rubric directive meant to address what we were looking for. 
Should I change back to sub-heading sir?

File: collect-settings.rst (hunk 5 above)
Comment (adammichaelwood) [reply collect-settings.rst]: in response to your question:

For this to be a rubric would suggest it is a part of the Import/Export Settings section.
This one should have stayed a heading, at the same level as Import/Export settings (~~~~~~~~)

The other ones make sense as rubrics, as they are consistent.

Comment (troublemagnet) [reply collect-settings.rst]: Oh this was an accident. sorry sir.
